### DISCUSSION TITLE
Forced Ragdoll after long fall

### DISCUSSION THREAD

**Discussion by djrobbnasty** [Sixze/ALS-Refactored#170] [Q&A] [answered]
Hi there,

I was hoping someone could point me in the right direction as to removing the forced ragdoll upon landing too hard.

Thanks

**Comment by kevytmaito11** [comment, marked as answer]
Go to ALS -> Data -> Character and open up CS_Default. Change "Start Ragdolling on Land" to false.

**Reply by djrobbnasty** [reply]
Thankyou
